frontend/app/src/App.test.tsx

```diff
@@ -931,24 +931,24 @@ describe("App.onHistoryChange", () => {
     pushStateSpy.mockRestore()
   })
 
-  it("does rerun when we are navigating to a different page and the last window history url contains an anchor", () => {
+  it("does rerun when we are navigating to a different page and the last window history url contains an anchor", async () => {
```

Comment:
This test wasn't actually running (`waitFor` returns a promise that wasn't awaited, so the expect statement inside wasn't running).
I made some changes based on my understanding of the test's intent, but perhaps @willhuang1997 could advise?

Reply:
The changes here LGTM -- I'm fairly sure the original test was just written slightly incorrectly, and the promise not being awaited was missed in code review